README.md

```diff
@@ -329,6 +329,13 @@ You can create a `.nvmrc` file containing version number in the project root dir
 
 For example, to make nvm default to the latest 5.9 release for the current directory:
 
+.nvmrc file:
```

Comment:
The following command *creates* the `.nvmrc` file with those contents - this block is redundant with the one after it.

What about that command could be made less confusing?